app/serializers/snippet_serializer.rb

```diff
@@ -1,4 +1,9 @@
-class UserSerializer
+class SnippetSerializer
   include JSONAPI::Serializer
-  attributes :id, :content
-end
+  attributes :id, :content, :user_id, :created_at, :updated_at
+  has_one :user
+
+  def user_id
+    current_user.id
```

Comment:
Why is this here?  `current_user` is a User object, so to get the id, you'd just do `current_user.id` in your code.